src/realm/views.cpp

```diff
@@ -310,6 +319,203 @@ DescriptorExport LimitDescriptor::export_for_handover() const
     return out;
 }
 
+IncludeDescriptor::IncludeDescriptor(Table const& table, std::vector<std::vector<LinkPathPart>> column_links)
```

Comment:
I think we normally put `const`first. A minor detail.
Could column_links not be a const & as well?